src/app/workflow/versions/versions.component.ts

```diff
@@ -50,6 +50,10 @@ export class VersionsWorkflowComponent extends Versions implements OnInit {
     return [4, 5];
   }
 
+  defaultSortColumn(): string {
```

Comment:
This probably can be made into a simple string like my-tool.component.ts's pageName

Reply:
Get warnings that abstract property can't be read from constructor so leaving it as is.

Reply:
https://github.com/dockstore/dockstore-ui2/compare/feature/2548/renameColumnInTagTable...feature/2458/suggestion